docs/devel/updating-deps.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+After having struggled getting the vendor packages upgraded I decided to give a
```

Comment:
I guess we can remove this "after having struggled" section :) Docs should be emotion-less.